src/CMakeLists.txt

```diff
@@ -1,5 +1,7 @@
-configure_file(version.h.in version.h @ONLY)
-include_directories(${CMAKE_CURRENT_BINARY_DIR})
+configure_file(version.h.in generated_sources/version.h @ONLY)
+
+add_library(version INTERFACE ${CMAKE_CURRENT_BINARY_DIR}/generated_sources/version.h)
```

Comment:
`add_library` of type `INTERFACE` with sources specified didn't appear in cmake until v3.19.  In our CI it always installs the latest cmake so it works there. On my box I was still using 3.18 so it barfed. I will be updating the README with an extra step about how to grab the latest (3.22+) from the kitware apt repo.

Reply:
That's a good point, it would also be possible to workaround the source in interfact by using target_sources or else. Let me know if you want this to be done.